cdk/domino_cdk/util.py

```diff
@@ -146,3 +153,60 @@ def overlay(alpha: dict, omega: dict) -> dict:
             return {}
         base_dict = check_type(dictionaries[0])
         return base_dict if len(dictionaries) == 1 else overlay(base_dict, cls.deep_merge(*dictionaries[1:]))
+
+    @staticmethod
+    def create_lambda(
```

Comment:
I feel like `provisioners/lambda/__init__.py` might be a better place for this? Everything else in here is non-provisioning-related helper functions (and mostly in service of `util.py` except for deep_merge), while this generates actual cdk lambda objects.

Reply:
Or if there was a separate-of-concerns thing, I could also see something like `domino_cdk/provisioners/lambda.py` for this and the lambda directory moved to `domino_cdk/lambda` making sense? It's not a big deal either way, just seems more categorically in line with the provisioners to me.

Reply:
I'm OK to move the function anywhere except __init__ of lambdas directory. lambdas directory contain files we upload to AWS. They are not a module, I'll remove accidental __init__ file to avoid confusion. 

The code to create lambda doesn't logically belong with the lambda bodies.

I'm OK to have one directory to hold all lambda bodies instead of having subdirectory along with every file using them.